CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ install(DIRECTORY ${PROJECT_SOURCE_DIR}/cmake/
 # install the scripts.
 # cmake-format: off
 install(PROGRAMS
+  scripts/run-clang-format
```

Comment:
clang is anyway used for C++ so why not call it cpp format in case another tool shows up?